plugins/obs-outputs/CMakeLists.txt

```diff
@@ -1,20 +1,21 @@
 project(obs-outputs)
 
-option(USE_SSL "Enable rtmps support with OpenSSL" OFF)
+option(WITH_RTMPS "Enable rtmps support with mbedTLS" OFF)
```

Comment:
I think this may be better if the default is `ON` for this option.  Facebook won't function without it if it's set to rtmps anyway.